clients/python/src/myanmartools/_params.py

```diff
@@ -1,80 +1,93 @@
-from importlib.resources import open_binary
-import numpy as np
+"""Helper functions for reading parameters of the model file."""
+
+from array import array
+from itertools import chain, repeat
 import struct
+from typing import BinaryIO, cast, Final, Iterator, Tuple
```

Comment:
Travis CI says:

```
    from typing import BinaryIO, cast, Final, Iterator, Tuple
ImportError: cannot import name 'Final' from 'typing' (/opt/python/3.7.1/lib/python3.7/typing.py)
```

Reply:
`typing.Final` is a feature added in Python 3.8, so we have to either drop support for Python 3.7 or avoid using the feature. I prefer the former because we have more static checking.

In fact, `typing.Iterator` and `typing.Tuple` are deprecated in Python 3.9. We eventually have to update for that and drop support for Python 3.8.